aggregators/codec.go

```diff
@@ -117,7 +116,7 @@ func (m *CombinedMetrics) UnmarshalBinary(data []byte) error {
 // ToProto converts ServiceAggregationKey to its protobuf representation.
 func (k *ServiceAggregationKey) ToProto() *aggregationpb.ServiceAggregationKey {
 	pb := aggregationpb.ServiceAggregationKeyFromVTPool()
-	pb.Timestamp = timestamppb.New(k.Timestamp)
+	pb.Timestamp = timestamppb.TimeToPBTimestamp(k.Timestamp)
```

Comment:
**question**: Is there a specific reason the aggregation key and other places are using time instead of switching to uint64 ?

Reply:
We are using some time.Time methods: some `time.Since()`, some `.Seconds()` and some `.String()`. I assume in memory it doesn't matter that much since time.Time is also just a simple struct.